yogstation/code/modules/antagonists/shadowling/shadowling_abilities.dm

```diff
@@ -175,6 +175,10 @@
 		if(istype(LO, /obj/machinery/power/floodlight))
 			var/obj/machinery/power/floodlight/FL = LO
 			FL.change_setting(2) // Set floodlight to lowest setting
+		if(istype(LO, /obj/item/stack/marker_beacon))
```

Comment:
```suggestion
		if(istype(LO, /obj/structure/marker_beacon))
```

Reply:
Also maybe turn this into a loop like glowshrooms as theos said if you want it to be bigger radius